internal/controller/envoy_gateway_extension_reconciler.go

```
@@ -41,7 +42,10 @@ import (
 
 // EnvoyGatewayExtensionReconciler reconciles Envoy Gateway EnvoyExtensionPolicy custom resources
 type EnvoyGatewayExtensionReconciler struct {
-	client *dynamic.DynamicClient
+	client         *dynamic.DynamicClient
```

golangci-lint (2.12.1)

[error] File is not properly formatted

(goimports)